osu.Game/Rulesets/Mods/ModFlashlight.cs

```diff
@@ -81,7 +81,7 @@ public virtual void ApplyToDrawableRuleset(DrawableRuleset<T> drawableRuleset)
 
         public abstract class Flashlight : Drawable
         {
-            internal BindableInt Combo;
+            protected internal BindableInt Combo { get; internal set; }
```

Comment:
okay if i change this to 

```csharp
            public readonly BindableInt Combo = new BindableInt();
```

and bind as we do in most places? feels weird as is

Reply:
Sure